src/components/Tabs/TabItem/TabItem.scss

```diff
@@ -1,49 +1,45 @@
 .TabItem {
-    background-color: var(--color-neutral-25);
-    border: {
-        color: var(--color-neutral);
-        style: solid;
-        top-left-radius: var(--border-radius);
-        top-right-radius: var(--border-radius);
-        width: var(--line-normal);
-        bottom-width: 0;
-    }
     color: var(--link-color-normal);
     cursor: pointer;
-    font: {
-        family: var(--font-family-primary);
-        size: var(--font-size-normal);
-    }
-    margin: {
-        top: var(--margin);
-        right: var(--margin);
-        left: var(--margin);
-    };
     outline: none;
     padding: var(--spacing-2x) var(--spacing-4x);
-    line-height: var(--line-height-normal);
     text-decoration: none;
-    transition: border-color var(--transition-duration);
+    transition: all var(--transition-duration);
+    text-align: center;
+    border-bottom: 4px solid transparent;
+    margin-bottom: -1px;
+    font-weight: var(--font-weight-bold);
 
-    &:hover {
-        border-color: var(--color-neutral-60);
+    &:hover,
+    &:focus {
+        border-bottom: 4px solid var(--color-neutral-25);
+        color: var(--link-color-hover);
     }
 
-    &:first-child {
-        margin-left: 0;
+    &--isActive,
+    &:active {
+        color: var(--color-foreground);
```

Comment:
I think this incorrect:
```
&:active {
 &:hover {}
}
```
It will be gerate CSS:
```.TabItem--20c4ZfdZd9:active:focus,.TabItem--20c4ZfdZd9:active:hover,.TabItem--isActive--1NIM9nd7JM```